modules/main-rules/pages/transportation.adoc

```diff
@@ -1,59 +1,62 @@
+== Transportation by {transport-x} units
 
-You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
-Obviously, once you disembark them then the normal rules apply again.
+:by-transport-x:
+include::partial$transportation-by-transport-units.adoc[]
 
-[NOTE]
-.Placement of disembarked units
-====
-When you disembark units, measure their movement from the base of the {transport-x} unit.
-If you do not have enough movement to clear the {transport-x} unit's base then place the disembarked unit in base contact with the {transport-x} unit.
-====
+:!by-transport-x:
 
-[NOTE.e40k]
-.Placement of disembarked units
-====
-We added this rule to minimise the effects that result from the assortment of different base sizes that players use.
-====
+== Transportation by {hitch-x} units
 
-== Destruction of {transport-x} units
+:by-hitch-x:
+include::partial$transportation-by-transport-units.adoc[]
 
-If a {transport-x} unit is destroyed then, just before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the unit suffers a hit.
-If the unit survives, place it in contact with the destroyed {transport-x} unit.
+:!by-hitch-x:
 
-Then remove the {transport-x} unit as normal.
+== Destruction of transportation units
 
-== Transportation by {hitch-x} units
+If a transportation unit is destroyed while cargo units are embarked then, before you remove the unit as a casualty, roll a D6 for each embarked unit: On a result of 1, 2 or 3, the embarked unit suffers a hit.
+If the embarked unit survives, place it completely within 5 cm of the destroyed transportation unit.
+
+Then remove the transportation unit as normal.
+
+[NOTE.e40k]
+.Destruction of transportation units
+====
+We make it clear that embarked units are subject to the same risk when their transportation unit is destroyed, regardless of which transportation ability is in effect.
+====
 
-Units that have the {hitch-x} ability can carry Infantry units across the battlefield -- they 'hitch a lift'.
-That is, they simply grab onto the outside of the vehicle and hang on for a crazy ride over the battlefield.
+== Transportation of units from another detachment
 
-To use the {hitch-x} ability to transport an Infantry unit:
+Where transportation units from one detachment carry units from another detachment, both detachments must be on the same orders.
+// IJW editing note - the war engine orders page will need a note saying how this works if you're *not* using war engine orders.
+// TODO: I wonder if we should thrash this detail out in the Experimental section? I mean, the original war engine orders rules were an afterthought and by no means universally accepted. Thus, I suggest that until such time as we nail down an alternative we should default to the original 'no orders' -- in which case this tract won't make sense here anyway.
```

Comment:
Adding a comment here to draw attention to these notes, for further discussion.

Reply:
I'm a bit confused here, that was the purpose of my note in the first place!

Reply:
Basically, my note proposes the opposite of yours — you're working from the assumption that we *have* war engine orders and that those rules are the default, whereas I'm working from opposite assumption, at least for now.
I started this 'conversation' thread so that we don't lose sight of this issue as we near the finish line on this branch (it's much easier to check for unresolved conversations than to hunt for comments on the source, in their various forms), and because we need somewhere to actually *have* the discussion.

Reply:
First off, I agree that this needs to be in a thread somewhere, and not just a comment in the source.

But I think we're still saying the same thing in different ways, when I said 'the war engine orders page' I was referring to whatever page discusses how orders interact with war engines. For example the Remastered version of https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/WarEngineMovement.htm

Reply:
We're talking at cross purposes, but I think I see where the misunderstanding lies. It sounds like in fact we're actually on the same page. So, how about this:

1. We remove the notes from the code here.
2. We move the empty `war-engine-orders.adoc` placeholder topic from the Special rules section to the Experimental rules section, and update the navigation.
3. We insert a note in `war-engine-orders.adoc`, to remind ourselves to address the point that you raised here. (Just to make sure that we don't overlook it when we get around to that topic.)

In that case we can then tidy up a little here.

However, there is still a fly in the ointment. Although we have this line, which covers detachments of regular units (#52):

>Where transportation units from one detachment carry units from another detachment, both detachments must be on the same orders.

...we don't actually mention what should happen with War Engine transportation units, which don't even *have* orders (yet). I'm inclined to ignore this for now, though I think that we should leave a TODO note as a reminder for us to revisit it at some point. We can also make this note visible so that others are aware. Or you might have an idea for a line we can stick in to clear it up properly now, and then we don't need a note.

```suggestion
{blank}[TODO: We should add something to address what should happen with War Engine transportation units, which don't even have orders (at least, not yet — we intend to experiment with new War Engine orders rules in due course).]
```

What do you think?

Reply:
> 1. We remove the notes from the code here.
> 2. We move the empty `war-engine-orders.adoc` placeholder topic from the Special rules section to the Experimental rules section, and update the navigation.
> 3. We insert a note in `war-engine-orders.adoc`, to remind ourselves to address the point that you raised here. (Just to make sure that we don't overlook it when we get around to that topic.)

Sounds good to me, and that's all my original note was referring to.


> However, there is still a fly in the ointment. Although we have this line, which covers detachments of regular units (#52):
> 
> > Where transportation units from one detachment carry units from another detachment, both detachments must be on the same orders.
> 
> ...we don't actually mention what should happen with War Engine transportation units, which don't even _have_ orders (yet). I'm inclined to ignore this for now, though I think that we should leave a TODO note as a reminder for us to revisit it at some point. We can also make this note visible so that others are aware. Or you might have an idea for a line we can stick in to clear it up properly now, and then we don't need a note.

OK, in that case we are still talking at cross-purposes. I wouldn't expect any mention of war engines on this page (apart from a Related information link), because I'm assuming that, like the Compendium, all the war engine exceptions are explained in the war engine section, not in the main rules.

Reply:
>I wouldn't expect any mention of war engines on this page (apart from a Related information link), because ... war engine exceptions are explained in the war engine section

Ah yes, I got side-tracked — one of the perils of getting sucked into particular details that are on a tangent. We are on the same page here.

Though we actually *don't* need the TODO note at `war-engine-orders.adoc` precisely because that *will* inherently integrate with the statement at line #52.

We need the rule/note somewhere else in the war engines section to cover the fact that the standard 'war engines don't have orders' rule *won't* integrate with line #52. I think I'll stick the reminder in `war engines.adoc` for now (alongside another reminder I've already drafted about {no-retreat} and {unbreakable}).

Perhaps in due course we'll want a specific topic of sub-heading like 'War Engine units as transportation'.

```suggestion
```